apps/token/src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en" class="bg-black w-full h-full">
+<html lang="en" class="dark bg-black w-full h-full">
```

Comment:
question (blocking): Do we need this on the token app? We aren't using the `dark:` prefix yet there. We do need to sort it out eventually though

Reply:
@mattrussell36 yeah we do need it so that the ui-toolkit components have the correct styling to match the dark look of the site. It's a good point and I guess we'll want a light mode for token eventually. I'll get a ticket made